docs/vscode.md

```diff
@@ -123,6 +136,32 @@ files = [
 ]
 ```
 
+### Secrets
```

Comment:
Do we need an explanation for what are "secrets"?

Reply:
Good call I can add a line or two.